client/app/pages/dashboards/dashboard.html

```diff
@@ -87,7 +87,7 @@ <h3>
   </div>
 
   <div class="m-b-10 p-15 bg-white tiled" ng-if="$ctrl.filters | notEmpty">
-    <filters filters="$ctrl.filters" on-change="$ctrl.filtersOnChange(filter, $modal)"></filters>
+    <filters filters="$ctrl.filters" on-change="$ctrl.filtersOnChange"></filters>
```

Comment:
Apparently, this does not update when adding/removing a filtered widget.
This bug happens in prod, but perhaps in this PR it's an ez fix?

Reply:
I'll check

Reply:
@ranbena It was an easy fix :smile: Done